### PR TITLE
fix links in commentary

### PR DESCRIPTION
**Pull request by mplscorwin**
Trivial documentation change fixing two broken links in commentary.  feel free to ignore if this doesn't make life easier :)

PS, I do have a copyright assignment on file with FSF for Emacs.

### CODE REVIEW

File: realgud-trepan-xpy.el [hunk 1]
```diff
@@ -27,7 +27,7 @@
 ;;; Commentary:
 
 ;; realgud support for the Python trepan-xpy
-;; See https://pypi.org/project/trepan-xpy/
+;; See https://pypi.org/project/trepan2/
```
Comment (rocky) [inline realgud-trepan-xpy.el]: The PyPI link is https://pypi.org/project/trepanxpy/

Sorry for the confusion.

Comment (rocky) [reply realgud-trepan-xpy.el]: If you correct the link, I'll merge and update ELPA. Thanks for doing this.

Comment (mplscorwin) [reply realgud-trepan-xpy.el]: Will do!

Comment (mplscorwin) [reply realgud-trepan-xpy.el]: Did and done!

**Review by rocky — APPROVED** [review]
LGTM - thanks.

**Pull request merged**